packages/field-plugin/vite.config.ts

```diff
@@ -13,7 +13,8 @@ export default defineConfig({
     lib: {
       // Could also be a dictionary or array of multiple entry points
       entry: fileURLToPath(new URL('./src/index.ts', import.meta.url)),
-      name: 'field-plugin',
+      name: 'FieldPlugin',
+      fileName: 'field-plugin',
```

Comment:
This isn't directly related to this PR, but I did it as a clean-up.

`name`: This is for a umd build. If user uses field plugin library with a umd build like

```html
<script src="...."></script>
```

Then they will be able to access the library like the following:

```js
window.FieldPlugin.createFieldPlugin(...)
```

That `FieldPlugin` is given by this `name` property.

And the `fileName` is literally the built output file name. Previously we only had `name` and it was `field-plugin`. Due to its casing, it was not correctly exposed to `window`.